modular_skyrat/master_files/code/datums/components/tippable.dm

```diff
@@ -0,0 +1,22 @@
+// Modular solution for alternative tipping visuals, for silicon/cyborgs.
+// Originally added by Skyrat-tg PR #10894
+/datum/component/tippable/set_tipped_status(mob/living/tipped_mob, new_status = FALSE)
+	// Defer to TG code if the mob isn't a silicon.
+	if (!istype(tipped_mob, /mob/living/silicon/robot))
```

Comment:
There is a define for that.
`iscyborg(tipped_mob)`

Reply:
Ok! I added it in commit f0277b0e4254fc49aa1e6c68396100d9f18178e7